Makefile

```diff
@@ -1,20 +1,26 @@
-TARGETS = naive cuda cupla kokkos oneapi
+TARGETS = naive cuda cupla kokkos oneapi alpakaCT alpakaRT
 
 .PHONY: all debug clean $(TARGETS)
 
+BOOST_BASE := /data/cmssw/slc7_amd64_gcc820/external/boost/1.67.0-pafccj/
+TBB_BASE := /data/cmssw/slc7_amd64_gcc820/external/tbb/2019_U3-pafccj
```

Comment:
Please use a generic `/usr/local/...` path, and move these below the definition of `ALPAKA_BASE` since (for now) only Alpaka depends on them.